### PR TITLE
Add uncrustify_install, uncrustify_flight Make targets

### PR DESCRIPTION
**Pull request by fujin**

### CODE REVIEW

**Comment by mlyle** [comment]
jenkins, test this please

File: uncrustify.cfg [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# first attempt at an uncrustify config file for dRonin
```
Comment (mlyle) [inline uncrustify.cfg]: can this please move to the make subdir?

**Comment by mlyle** [comment]
Works great on OSX- have you confirmed on Linux?

File: make/tools.mk [hunk 2]
```diff
@@ -439,6 +439,45 @@ astyle_clean:
 	$(V0) @echo " CLEAN        $(ASTYLE_BUILD_DIR)"
 	$(V1) [ ! -d "$(ASTYLE_BUILD_DIR)" ] || $(RM) -r "$(ASTYLE_BUILD_DIR)"
 
+# Set up uncrustify tools
+UNCRUSTIFY_DIR := $(TOOLS_DIR)/uncrustify-0.61
+UNCRUSTIFY_BUILD_DIR := $(DL_DIR)/uncrustify
+libusb_win_install: LIBUSB_WIN_FILE := libusb-win32-bin-1.2.6.0.zip
```
Comment (mlyle) [inline make/tools.mk]: was this erroneously included?

Comment (fujin) [inline make/tools.mk]: yep that's an error

File: uncrustify.cfg (hunk 1 above)
Comment (fujin) [inline uncrustify.cfg]: yup

**Comment by fujin** [comment]
@mlyle yep worked on Linux

**Pull request closed without merging**